homeassistant/components/asuswrt/device_tracker.py

```diff
@@ -55,41 +52,22 @@ def add_entities(router, async_add_entities, tracked):
 class AsusWrtDevice(ScannerEntity):
     """Representation of a AsusWrt device."""
 
+    _attr_should_poll = False
+
     def __init__(self, router: AsusWrtRouter, device) -> None:
         """Initialize a AsusWrt device."""
         self._router = router
         self._device = device
-
-    @property
-    def unique_id(self) -> str:
-        """Return a unique ID."""
-        return self._device.mac
-
-    @property
-    def name(self) -> str:
-        """Return the name."""
-        return self._device.name or DEFAULT_DEVICE_NAME
+        self._attr_unique_id = self._device.mac
+        self._attr_name = self._device.name or DEFAULT_DEVICE_NAME
+        self._attr_source_type = SOURCE_TYPE_ROUTER
+        self._attr_mac_address = self._device.mac
```

Comment:
I don't think those `_attr_`s exist yet. 

`mac_address` returns None in ScannerEntity.

https://github.com/home-assistant/core/blob/aa022d4c526421effdd6ac2d4bd54bad9168085e/homeassistant/components/device_tracker/config_entry.py#L145-L148

We should add support for entity class attributes to `internal` integrations first.